tests/rule-engine/fc-config.scm

```diff
@@ -1,10 +1,21 @@
 ;
 ; Configuration file for the example crisp rule base system (used by
 ; fc.scm)
-;
-; Before running any inference you must load that file
 
-; Load the rules
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Load required modules and utils ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(use-modules (opencog))
+(use-modules (opencog rule-engine))
+
+(load-from-path "utilities.scm")
+(load-from-path "av-tv.scm")
```

Comment:
It will stay true -- the base opencog guile module provides the various common utilities, and I see no reason to change this, other than to periodically go in there and clean up whatever mess and cruft accumulates over time.

Anyway, this is very low priority, I just thought I'd mention it.